starters/next-wordpress-starter/components/header.jsx

```diff
@@ -7,8 +7,8 @@ export default function Header() {
 				<ul className="flex flex-row flex-wrap sm:flex-nowrap list-none justify-between max-w-screen-sm mx-auto">
 					{[
 						['🏠 Home', '/'],
-						['📰 Posts', '/posts'],
-						['📑 Pages', '/pages'],
+						['📰 Posts', '/posts/1'],
```

Comment:
Ideally we don't want to hardcode the links here to include `/1`. `/posts` and `/pages` should be valid paths that are not redirected. The Drupal starter example allows `/examples/pagination` and does not redirect to `/examples/pagination/1` but this implementation automatically redirects to `/1` for some reason.

Reply:
Ah I see more clearly now, this has to do with with routing being enabled on the pager, I'll disable that feature to honor the `/pages` route.